tests/app/user/test_rest.py

```diff
@@ -731,3 +731,38 @@ def test_get_orgs_and_services_only_shows_users_orgs_and_services(admin_request,
             }
         ]
     }
+
+
+def test_search_for_users_by_email(notify_db, client):
+    user_1 = create_user(email='findel.mestro@foo.com')
```

Comment:
Code style is failing because these two variables are not used. 

Just create the user? create_user(email='me.ignorra@foo.com') etc

Reply:
👍